my_locustfiles/application_store.py

```diff
@@ -0,0 +1,35 @@
+from locust import HttpUser, task
+from common.config import Configs
```

Comment:
common doesn't contain a `__init__.py` file, does this import ok?

The __init__.py file allows you to access a python file from anywhere in the folder structure, because it registers a folder with python as a local package.